color/src/color.rs

```diff
@@ -426,6 +434,15 @@ impl<CS: ColorSpace> PremulColor<CS> {
         Self { components, cs }
     }
 
+    /// Split out the opaque components, discarding the alpha.
+    ///
+    /// This is a shorthand for un-premultiplying the alpha and
+    /// calling [`AlphaColor::discard_alpha`].
```

Comment:
I think I might mention that the result of this method on a fully transparent color is black.

Reply:
Added.